deploy/charts/rook-ceph/values.yaml

```diff
@@ -278,17 +279,17 @@ csi:
   #rbdLivenessMetricsPort: 9080
   #kubeletDirPath: /var/lib/kubelet
   #cephcsi:
-    #image: quay.io/cephcsi/cephcsi:v3.6.0
+  #image: quay.io/cephcsi/cephcsi:v3.6.0
   #registrar:
-    #image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.5.0
+  #image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.5.0
```

Comment:
This indentation should not be changed, your editor must have changed this

Reply:
it is the editor. should be good now